### ISSUE TITLE
Attribute order for onclick event attributes

### ISSUE THREAD

**Issue by vanillajonathan** [mdo/code-guide#259]
The [HTML attribute order](https://codeguide.co/#html-attribute-order
) section does not include the order of event attributes such as `onclick`, `onblur`, `onchange`, etc.

**Comment by msantana-dev** [comment]
Might be due to usage of event attributes being discouraged.
[From MDN:](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Events#inline_event_handlers_%E2%80%94_dont_use_these)
> You can find HTML attribute equivalents for many of the event handler properties; however, you shouldn't use these — they are considered bad practice. It might seem easy to use an event handler attribute if you are doing something really quick, but they quickly become unmanageable and inefficient.

**Comment by vanillajonathan** [comment]
Yes, but when you use an SPA such as [React](https://reactjs.org/docs/handling-events.html) you would use event handlers such as `onClick`, `onBlur`, `onChange`, etc, or with [Vue](https://vuejs.org/guide/essentials/event-handling.html) `@click`, `@blur`, `@change`, or with [Angular](https://angular.io/guide/event-binding) `(click)`, `(blur)`, `(change)`.
